_docs/concepts/what-is-grafeas/overview.md

```diff
@@ -1,223 +1,111 @@
 ---
 title: Overview
 overview: Provides a conceptual introduction to Grafeas, including the problems it solves and its high-level architecture.
-          
+
 order: 15
 
 layout: docs
 type: markdown
 ---
 
-This document introduces Grafeas: a common API and language to store, query and retrieve metadata about software components.
-
-Grafeas defines metadata API spec for computing components (e.g., VM images, container images, jar files, scripts) that can assist with aggregations over your metadata. Grafeas uses two API concepts, a **note** and an **occurrence**. This division allows 3rd party metadata providers to create and manage metadata on behalf of many customers. Additionally, the division also allows implementation of access control settings that allow fine grain access control.
-
-## Definition of terms
-**Notes**: A note is an item or condition that can be found via an analysis or something that is used multiple times in a process. For example, a CVE could be the result of a vulnerability analysis of a Linux package. In a build process, we would store information about our builder in a note. 
+# Grafeas: A Component Metadata API
 
-A note name should take the format `/projects/<project_id>/notes/<note_id>` where the project_id would typically be different from the project where the occurrence is created and the note_id would be unique per note-project, and informative if possible. 
+Grafeas defines an API spec for managing metadata about software resources, such
+as containers, Virtual Machine (VM) images, JAR files, and scripts. You can use
+Grafeas to define and aggregate information about your project's components.
 
-Access to notes should be read-only for users who have access to occurrences referencing them, and editable only by the note owner.
+Grafeas divides metadata information into [_notes_](#notes), which are
```

Comment:
How about splitting this sentence into two? 

Grafeas divides metadata information into [_notes_](#notes) and [_occurrences_](#occurrences). Notes are ...